### PR TITLE
add github repos to footer

### PR DESCRIPTION
**Pull request by terrazoon**
## Description

Add GitHub repos to footer.

## Security Considerations

N/A

### CODE REVIEW

**Comment by terrazoon** [comment]
Screenshot 2024-07-17 at 8 57 25 AM

File: app/templates/new/components/footer.html [hunk 1]
```diff
@@ -58,6 +58,18 @@ <h2 class="usa-sr-only">Support links</h2>
...
       </div>
     </nav>
+    <section class="usa-identifier__section usa-identifier__section--usagov"
+      aria-label="Github Repos">
+      <div class="usa-identifier__container">
+        <div class="usa-identifier__required-links-item">
+          Find us on Github:
+        </div>
+        <a href="https://github.com/gsa/notifications-admin" class="usa-identifier__required-link">Notify.gov Admin repo</a>
+        <a href="https://github.com/gsa/notifications-api" class="usa-identifier__required-link">Notify.gov API repo</a>
```
Comment (ccostino) [inline app/templates/new/components/footer.html:68]: Let's turn this into a bulleted list for better readability/formatting:

```suggestion
        <ul>
          <li><a href="https://github.com/gsa/notifications-admin" class="usa-identifier__required-link">Notify.gov Admin repo</a></li>
        <li><a href="https://github.com/gsa/notifications-api" class="usa-identifier__required-link">Notify.gov API repo</a></li>
      </ul>
```

**Review by ccostino — CHANGES_REQUESTED** [review]
Thanks, @terrazoon!  One small request to format the links in a list.

**Review by ccostino — APPROVED** [review]
Thanks, @terrazoon!

**Pull request merged**